buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -3,31 +3,29 @@ object Versions {
     const val project = "0.4.2-SNAPSHOT"
 
     const val java = 17
-    const val kotlin = "1.9.10"
+    const val kotlin = "1.9.22"
     const val wrapper = "8.4"
 
     const val radarCommons = "1.1.2"
     const val confluent = "7.6.0"
     const val kafka = "$confluent-ce"
     const val avro = "1.11.3"
 
-    const val managementPortal = "2.0.0"
+    const val managementPortal = "0.8.3"
```

Comment:
I think this should be `2.0.0`?

Reply:
You're right, though there is a bit of a strange situation going on there (0.8.3 is newer but it branched off of 0.8.2 and I did not independently update oauth-client-util)

If we ever make 0.8.4 it would be good to get all dependencyversions into the same file.

Anyhoo, reverted